azure-cron.yml

```diff
@@ -177,10 +177,9 @@ jobs:
             echo "Publishing $GITHUB to VSCode Marketplace"
             git checkout $GITHUB
             cd compiler/daml-extension
-            yarn
             # This produces out/src/extension.js
-            yarn run compile
-            vsce publish ${GITHUB#v} -p $MARKETPLACE_TOKEN
+            bazel run @nodejs//:bin/yarn
```

Comment:
I think we still need the `yarn compile` step.

Reply:
Good point, will add that.